src/cargo.js

```diff
@@ -82,3 +93,11 @@ export const handleCargo = async function (self, result) {
 
   return {wasmFile, jsFile};
 };
+
+const makeCargoError = result => {
+  const tmpl = `Cargo encountered an error while compiling your crate
+${result.stderr}`;
```

Comment:
We probably don't need to interpolate the `stderr` into the error message since we already report the compilation errors via Webpack separately.